crates/bench_utils/src/utils.rs

```diff
@@ -0,0 +1,140 @@
+use std::str::FromStr;
+
+use if_chain::if_chain;
+use proc_macro::TokenStream;
+use proc_macro2::{Ident, Span, TokenStream as TokenStream2};
+use quote::{quote, ToTokens};
+
+pub fn parse_expr<T: FromStr>(expr: syn::Expr) -> T {
+  expr.to_token_stream().to_string().parse().ok().unwrap()
+}
+
+pub struct ArrayAssign {
+  pub var_name: syn::Expr,
+  pub var_val: syn::Expr,
+  pub num_locations: usize,
+}
+
+/// Parses the expression:
+/// <pre>
```

Comment:
I would just use Markdown syntax instead of HTML.

Reply:
HTML allowed parts of the code block to be bold but I agree it's probably best to just use Markdown